storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/operations/delete/DeleteByQueryStub.java

```diff
@@ -86,4 +87,17 @@ public void call(DeleteResult deleteResult) {
                 })
                 .checkBehaviorOfObservable();
     }
+
+    void verifyBehavior(@NonNull Single<DeleteResult> deleteResultSingle) {
```

Comment:
In many cases `verifyBehavior` for `Observable` also checks that `observeChangesInTables` should be called, but it is wrong behavior for `Single`. So for the sake of consistency, I decided to implement distinct `verifyBehavior` for `Single`.